.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Make a player with a gun, magazine, and some ammo.
+2. Try to aim at a wall.
+3. There's no way to increase aim level.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+Ideally also describe *why* you expect it to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Versions and configuration(please complete the following information):**
+ - OS: [e.g. iOS 8 or Windows 10]
+ - Game Version [e.g. ]
+ - Graphics version [Tiles or Terminal]
+ - Mods loaded [e.g.dda, boats, hacktheplanet, StatsThroughSkills  ]
+
+**Additional context**
+Add any other context about the problem here.
```

Comment:
`about the problem here.  Additionally, if you have a save prior to a bug occurring and it is reproducible, you can also upload a .zip of your save file here.`